sopel/bot.py

```diff
@@ -48,6 +48,38 @@ def nop(bot, cap):
         self.success = success or nop
 
 
+class _ServerInfo(object):  # Eventually this can become a Python 3 data class
+    """A data class for server information.
+
+    capabilities: A dict mapping supported IRCv3 capabilities to their options.
+
+        For example, if the server specifies the capability
+        ``sasl=EXTERNAL``, it will be here as ``{"sasl": "EXTERNAL"}``.
+        Capabilities specified without any options will have ``None``
+        as the value.
+
+        For servers that do not support IRCv3, this will be an empty
+        set.
```

Comment:
Since I moved the `bot.server_capabilities` object into the  `_ServerInfo` class, I just copy-pasted the existing docstring, which has the error you reference. Since the new ISUPPORT class does not touch the `server_capabilties`, this error still exists (as it has for a long time).